src/forms/FormikSwitchField.stories.tsx

```diff
@@ -0,0 +1,55 @@
+import { Form, Formik } from 'formik'
+import React from 'react'
+import FormikSwitchField from './FormikSwitchField'
+
+export const Default = () => {
+  return (
+    <div>
+      <Formik
+        initialValues={{
+          name: '',
+        }}
+        isInitialValid={false}
+        onSubmit={() => {
+          console.log('nothing to submit')
```

Comment:
(optional) - maybe log the current value of the switch to show that it's working or use something like alert() to highlight its functionality in ladle right away?

Reply:
Done :)

Reply:
Great, thanks! Would is maybe also make sense to drop certain attributes that will not be used in connection with the Switch used as a field, like the `fluid` prop or maybe even `labelLeft`?